cves/CVE-2014-0113.yml

```diff
@@ -205,4 +209,6 @@ mistakes:
     Use those questions to inspire your answer. Don't feel obligated to answer
     every one. Write a thoughtful entry here that those in the software
     engineering industry would find interesting.
-  answer:
+  answer: This seems like an inherent design mistake when the CookieInterceptor 
+    class was created. You have to keep a clear idea of permissions to possibly
+    private methods when writing code.
```

Comment:
What mitigations does the CWE give for this vulnerability? What can be done to prevent this mistake in the future?

Reply:
A mitigation, besides updating to 2.3.16.2, is replacing the previous class related pattern with '(.*\.|^|.*|\[('|"))class(\.|('|")]|\[).*' on the list of excludeParams. In the future, testing permissions as well as the code being effected could help to mitigate a possible repeat of the mistake.